runtime/src/main/java/org/corfudb/protocols/CorfuProtocolLogData.java

```diff
@@ -0,0 +1,59 @@
+package org.corfudb.protocols;
+
+import com.google.protobuf.ByteString;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.protocols.wireprotocol.LogData;
+import org.corfudb.runtime.proto.LogData.LogDataMsg;
+import org.corfudb.runtime.proto.LogData.ReadResponseMsg;
+
+/**
+ * This class provides methods for creating and converting between the Protobuf
+ * objects defined in log_data.proto and their Java counterparts. Used by the
+ * LogUnit RPCs.
+ */
+@Slf4j
+public class CorfuProtocolLogData {
+    // Prevent class from being instantiated
+    private CorfuProtocolLogData() {}
+
```

Comment:
You can mark the class as final to prevent it from being inherited as well since its just a utils class.

Reply:
Looks like we should mark all the CorfuProtocol*.java files to be final?

Reply:
Done.